grafana-plugin/src/components/WithContextMenu/WithContextMenu.tsx

```diff
@@ -59,4 +63,8 @@ export const WithContextMenu: React.FC<WithContextMenuProps> = ({
       )}
     </div>
   );
+
+  function isContextMenuOpen() {
+    return isOpen === undefined ? isMenuOpen : isOpen;
+  }
```

Comment:
This is very confusing :D We have now `isMenuOpen`, `forceIsOpen` and `isOpen` (that is another way of forcing opening/closing). If we're not able to simplify that in scope of this PR, can we create an issue right away to simplify that and put it in the sprint backlog to avoid longlasting TODOs?

Reply:
I'll revert these changes, these were some experiments in attempting to use this component instead